### ISSUE TITLE
Cotary Create: Feedback February 21st - deployed live at https://daocreator-v2.herokuapp.com

### ISSUE THREAD

**Issue by mrrobot16** [dOrgTech/DAOcreator#419]
Please write down all feedback that you can come for https://daocreator-v2.herokuapp.com/

For stylings feedback can be checked here https://github.com/dOrgTech/DAOcreator/issues/408

Thanks :)

**Comment by orishim** [comment]
Hours field looks messed up in Advanced view
<img width="155" alt="Screen Shot 2020-02-21 at 2 21 38 PM" src="https://user-images.githubusercontent.com/10736693/75072876-94c56680-54b5-11ea-9bb4-e0f57fc0822e.png">

Also these should not have AM or PM, as they are just a number of hours in addition to the number of days.

Overall it is pretty difficult and glitchy trying to edit the days/hours values.

**Comment by orishim** [comment]
I think having "Connect to Web3" in the Members step is awkward/confusing.. either it should be at the beginning and we include the user by default as before, or we should move it to the deploy step and not add the user by default.

I am personally in favor of the first option.

**Comment by dOrgJelli** [comment]
When I export the config, "tokenName" is set to "0"
![image](https://user-images.githubusercontent.com/5522128/75077268-3b1e6580-54d0-11ea-8eb1-bee5a9dda2cd.png)

**Comment by dOrgJelli** [comment]
Layout on mobile isn't correct
![image](https://user-images.githubusercontent.com/5522128/75077354-7c167a00-54d0-11ea-974c-5fe5e70d7eca.png)

**Comment by dOrgJelli** [comment]
In the "Advanced Configuration", why does it say "Deploy ${Scheme Name}"
![image](https://user-images.githubusercontent.com/5522128/75077479-e92a0f80-54d0-11ea-8981-5c00d731de5f.png)

**Comment by dOrgJelli** [comment]
These input fields are strange to me. Does this mean "30 days & 12 hours"? Maybe have 3 separate input boxes for days, hours, and minutes?

![image](https://user-images.githubusercontent.com/5522128/75077593-41611180-54d1-11ea-816d-b38114e58875.png)

**Comment by dOrgJelli** [comment]
Some of these boxes have up and down arrows when you select them, and some do not
![image](https://user-images.githubusercontent.com/5522128/75077653-66558480-54d1-11ea-80a8-82a8ba9938f9.png)

**Comment by dOrgJelli** [comment]
(1) I miss having the blockie that takes you to etherscan if you click on it. Personal opinion though, would leave it up to @pimato to make the UX decision.
(2) The input field for this box shows a red curly under my address when I edit it. There should be a way to disable spell check for an input field right?
![image](https://user-images.githubusercontent.com/5522128/75077774-b46a8800-54d1-11ea-8e9f-10c9f6085df7.png)

**Comment by dOrgJelli** [comment]
When you edit the advanced configuration, the speed toggles turn to grey which is nice, but as a new user I might not understand why this is. I think what might help this is to have a warning popup when a user selects the "disabled" speed toggle, letting them know "your configuration will be overwritten with the present" or something like that. This way the user can say "oh no don't do that, I want to keep my custom config.
![image](https://user-images.githubusercontent.com/5522128/75077939-30fd6680-54d2-11ea-84cb-584f76b76aaa.png)

**Comment by dOrgJelli** [comment]
I feel like these boxes should be yellow instead of red, as they're warnings after all. As a user, when I saw this red dialog popup I thought the deployment failed:
![image](https://user-images.githubusercontent.com/5522128/75078220-0fe94580-54d3-11ea-86ab-45dd78b823da.png)

**Comment by dOrgJelli** [comment]
When you resume a DAO configuration after reloading the page, the toggles are changed back to enabled, even if the user has made changes in the advanced page:
![image](https://user-images.githubusercontent.com/5522128/75078435-94d45f00-54d3-11ea-964f-a547648f620a.png)

**Comment by dOrgJelli** [comment]
If something goes wrong, it'd be great to allow the user to export a log of what happened. We can do this by [storing all logged lines](https://github.com/dOrgTech/DAOcreator/blob/320096959ad914106f51c8d2c8693b1511472662/packages/ui_v2/src/components/commonV2/dao/Migrator/index.tsx#L113) and when the user requests, writing all of these to a file. The V1 UI supports this and could be a good place to look for inspiration.
![image](https://user-images.githubusercontent.com/5522128/75078653-293ec180-54d4-11ea-8bc1-2f0faf4f9ac7.png)

**Comment by Flash-Git** [comment]
### Replying to some of the above feedback. Not to be confused with a comprehensive action plan!

> 
> 
> Hours field looks messed up in Advanced view
> <img alt="Screen Shot 2020-02-21 at 2 21 38 PM" width="155" src="https://user-images.githubusercontent.com/10736693/75072876-94c56680-54b5-11ea-9bb4-e0f57fc0822e.png">
> 
> Also these should not have AM or PM, as they are just a number of hours in addition to the number of days.
> 
> Overall it is pretty difficult and glitchy trying to edit the days/hours values.

Weird, I'm not getting the AM/PM: ![image](https://user-images.githubusercontent.com/20420828/75083931-e9e89300-551c-11ea-9269-82b95c7a8fd4.png)

_____

> 
> 
> I think having "Connect to Web3" in the Members step is awkward/confusing.. either it should be at the beginning and we include the user by default as before, or we should move it to the deploy step and not add the user by default.
> 
> I am personally in favor of the first option.

The feedback we got before was to only introduce metamack popups when they're required as they can put people off. See [#352](https://github.com/dOrgTech/DAOcreator/issues/352#issuecomment-581002697):
"I don't like having to log into Metamask when the dApp opens. I feel it should happen when it's first needed. For example, when the users gets to the Members step, have a button for "Connect Wallet", and then once a wallet is connected change that button to "Add Self" or something like that which populates the member field with your address."
What do you think @dOrgJelli, @pimato?
_____

> 
> 
> Layout on mobile isn't correct
> ![image](https://user-images.githubusercontent.com/5522128/75077354-7c167a00-54d0-11ea-974c-5fe5e70d7eca.png)

I previously created an issue about this here: #356 

_____

> 
> 
> When you edit the advanced configuration, the speed toggles turn to grey which is nice, but as a new user I might not understand why this is. I think what might help this is to have a warning popup when a user selects the "disabled" speed toggle, letting them know "your configuration will be overwritten with the present" or something like that. This way the user can say "oh no don't do that, I want to keep my custom config.
> ![image](https://user-images.githubusercontent.com/5522128/75077939-30fd6680-54d2-11ea-84cb-584f76b76aaa.png)

Great idea!
_____

> 
> 
> (1) I miss having the blockie that takes you to etherscan if you click on it. Personal opinion though, would leave it up to @pimato to make the UX decision.

Looks like the blockies still take you Etherscan. Do you mean there should be one in the input field?

____

> 
> 
> I feel like these boxes should be yellow instead of red, as they're warnings after all. As a user, when I saw this red dialog popup I thought the deployment failed:
> ![image](https://user-images.githubusercontent.com/5522128/75078220-0fe94580-54d3-11ea-86ab-45dd78b823da.png)

I'll sneak this into #416 
_______

> 
> 
> When you resume a DAO configuration after reloading the page, the toggles are changed back to enabled, even if the user has made changes in the advanced page:
> ![image](https://user-images.githubusercontent.com/5522128/75078435-94d45f00-54d3-11ea-964f-a547648f620a.png)

I will take care of this.
______


> 
> 
> If something goes wrong, it'd be great to allow the user to export a log of what happened. We can do this by [storing all logged lines](https://github.com/dOrgTech/DAOcreator/blob/320096959ad914106f51c8d2c8693b1511472662/packages/ui_v2/src/components/commonV2/dao/Migrator/index.tsx#L113) and when the user requests, writing all of these to a file. The V1 UI supports this and could be a good place to look for inspiration.
> ![image](https://user-images.githubusercontent.com/5522128/75078653-293ec180-54d4-11ea-8bc1-2f0faf4f9ac7.png)

Yep, working on it here: #416.

**Comment by dOrgJelli** [comment]
@Flash-Git I agree, I like having web3 connection done as late as possible. Maybe instead of having a button in the member section say "connect web3", have it say "add your wallet"?

**Comment by dOrgJelli** [comment]
@Flash-Git Re: blockies, yah I think it was super nice in the old one where the blocky was in the input field of the address :P

![image](https://user-images.githubusercontent.com/5522128/75084744-3bc6f400-54f0-11ea-861f-780796fb7e1e.png)

**Comment by dOrgJelli** [comment]
@Flash-Git Re: AM/PM this is strange, maybe it's browser specific. Maybe we just make the text field a normal text field, and add the HR & MIN text ourselves?

**Comment by Flash-Git** [comment]
> 
> 
> @Flash-Git I agree, I like having web3 connection done as late as possible. Maybe instead of having a button in the member section say "connect web3", have it say "add your wallet"?

> 
> 
> @Flash-Git Re: blockies, yah I think it was super nice in the old one where the blocky was in the input field of the address :P
> 
> ![image](https://user-images.githubusercontent.com/5522128/75084744-3bc6f400-54f0-11ea-861f-780796fb7e1e.png)

On it! I will create a pr for these and other small changes that come up from this.

**Comment by mrrobot16** [comment]
> In the "Advanced Configuration", why does it say "Deploy ${Scheme Name}"
> ![image](https://user-images.githubusercontent.com/5522128/75077479-e92a0f80-54d0-11ea-8981-5c00d731de5f.png)

@dOrgJelli Its because we change the copy in the library so its now rendering "Funding and..." in that part what shall we say? "Contribution Reward"?

**Comment by mrrobot16** [comment]
> Some of these boxes have up and down arrows when you select them, and some do not
> ![image](https://user-images.githubusercontent.com/5522128/75077653-66558480-54d1-11ea-80a8-82a8ba9938f9.png)

Shall all number fields have the arrows right?

**Comment by mrrobot16** [comment]
> When I export the config, "tokenName" is set to "0"
> ![image](https://user-images.githubusercontent.com/5522128/75077268-3b1e6580-54d0-11ea-8eb1-bee5a9dda2cd.png)

@dOrgJelli tokenName should be equal to DAOSymbol correct?

**Comment by dOrgJelli** [comment]
@mrrobot16
- tokenName should be equal to orgName
- No I mean the "Deploy ..." doesn't quite make sense to me. Why is the "Deploy" word there? Maybe reword it to "Enable ${scheme_name}?"

**Comment by orishim** [comment]
I think we had said tokenName should be equal to orgName + "Token"

So orgName = "dOrg" means tokenName = "dOrg Token"

**Comment by Flash-Git** [comment]
> I think we had said tokenName should be equal to orgName + "Token"
> 
> So orgName = "dOrg" means tokenName = "dOrg Token"

Done #429

**Comment by dOrgJelli** [comment]
@mrrobot16 
I personally don't like the arrows :P

In V1, we just have number fields as normal text fields, and an error is thrown if it isn't a valid number
